### PR TITLE
Fix/5328 settings datenspende pic missing text

### PR DESCRIPTION
**Pull request by 30mar**
## Description
the are changes in the screen compared to the figma design

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-5328

### CODE REVIEW

File: src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/SettingsDataDonationViewModel.swift [hunk 1]
```diff
@@ -25,8 +25,21 @@ final class SettingsDataDonationViewModel: BaseDataDonationViewModel {
...
 			$0.add(
 				.section(
+					header: .image(
+						UIImage(named: "Illustration_Datenspendedatenspende"),
+						accessibilityLabel: AppStrings.NotificationSettings.imageDescriptionOn,
+						accessibilityIdentifier: AccessibilityIdentifiers.NotificationSettings.DeltaOnboarding.imageOn,
+						height: 250
+					),
 					cells: [
-						.headline(text: AppStrings.DataDonation.Info.introductionText),
```
Comment (pascal-brause) [inline src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/SettingsDataDonationViewModel.swift:29]: Is `AppStrings.DataDonation.Info.introductionText` still used somewhere? if not, we could remove the link in the app strings and perhaps in the localization file, too.

File: src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift [hunk 2]
```diff
@@ -14,7 +14,7 @@ class DataDonationViewController: DynamicTableViewController, DeltaOnboardingVie
 		largeTitleDisplayMode: UINavigationItem.LargeTitleDisplayMode = .never
 	) {
 		self.viewModel = viewModel
-		self.largeTitleDisplayMode = largeTitleDisplayMode
+		self.largeTitleDisplayMode = .never
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift:17]: The parameter is still there, so it seems confusing to me that it is not used

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift (hunk 2 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift:17]: The DataDonationViewController is initialized also in the on boarding, im confirming with Kerstin if it should change as well

File: src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/SettingsDataDonationViewModel.swift (hunk 1 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewModels/SettingsDataDonationViewModel.swift:29]: it is used in another place but i will confir, im confirming with Kerstin if it should change as well

File: src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift (hunk 2 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift]: I remove the variable and also added an extra space to the legal text in the on boarding so we match the figma design

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Review by pascal-brause — APPROVED** [review]

**Review by nickguendling — APPROVED** [review]

**Pull request merged**